lib/pure/logging.nim

```diff
@@ -150,8 +150,11 @@ method log*(logger: ConsoleLogger, level: Level, args: varargs[string, `$`]) =
       {.emit: "console.log(`cln`);".}
     else:
       try:
-        writeLine(stdout, ln)
-        if level in {lvlError, lvlFatal}: flushFile(stdout)
+        if level in {lvlError, lvlFatal, lvlWarn}:
```

Comment:
I think ALL log levels should go to `stderr` by default

Reply:
updated (y)